src/_help/specification-support/asyncapi-support.md

```diff
@@ -57,3 +57,11 @@ To describe these authentication types, please use our [custom `x-topics` proper
 ## readOnly and writeOnly properties
 
 JSON Schema provides the possibility to declare a property as read or write only. Read more in [the JSON Schema section of this documentation](/help/specification-support/json-schema#readonly-and-writeonly-properties).
+
+You can add extra information to your documentation by using Bump.sh custom `x-topics`.
+
+Read more in the [Topics section of this documentation](/help/doc-topics).
+
+## Expose your beta features (`x-beta`)
+
+This custom property allows you to identify some components of your documentation as beta. Find out more in our [dedicated section](/help/specification-support/doc-beta).
```

Comment:
This should be moved to the new `specification-support/extensions.md` file

Reply:
Not sure, we have the same text in openapi-support.md. We should keep both or remove both.

Reply:
Just seeing the next comment. We remove both ?

Reply:
Yes we remove both from the Spec-specific files (OpenAPI and AsyncAPI) to move the paragraph in the generic `specification-support/extensions.md` file.